orchestra/actions/install.py

```diff
@@ -112,6 +112,7 @@ def _update_metadata(self, file_list, install_time, source, set_manually_insalle
         save_metadata(metadata, self.config)
 
     def _prepare_tmproot(self):
+        # fmt: off
```

Comment:
This is not very elegant and will become pervasive.
Also, what formatter is this? Is it automated or is it just your IDE? Other devs don't use fancy IDEs.

Maybe your formatter threats specially more *standard* ways of dedenting?
[Here](https://stackoverflow.com/questions/2504411/proper-indentation-for-python-multiline-strings) they suggest `textwrap.dedent` and others.

Reply:
Almost all those annotations are placed around `dedent`ed strings, what do you mean?
Anyway, they are interpreted by black (there are info on how to set it up as a pre-commit hook in this PR) and they cause it to leave the enclosed sections as-they are. I reviewed how black re-indents those and you're right, it's pointless to tell black not to format them in almost all cases (one notable exception for example being the regexes in `orchestra/util.py` which would be collapsed on one line). I'll remove the annotations and reindent the files.